hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeResolverTest.kt

```diff
@@ -73,6 +75,7 @@ class AttributeResolverTest {
   fun `should resolve attributes`(attributeName: String, attribute: Attribute) {
     assertThat(attributes[attributeName]?.type).isEqualTo(attribute.type)
     assertThat(attributes[attributeName]?.openSearchName).isEqualTo(attribute.openSearchName)
+    assertThat(attributes[attributeName]?.nested).isEqualTo(attribute.nested)
```

Comment:
Tests that we can correctly derive whether an attribute is part of a nested object or not.